syntax = "proto3";
package YL.Login;
import "YLBase.proto";

//请求消息服务器
message YLMsgServerRequest
{
    string s = 1;
}

//请求响应
message YLMsgServerRespone
{
    YL.Base.ResultType result_code = 1;
    string prior_ip = 2;    //最优ip
    string back_ip  = 3;    //备份ip
    uint32 port     = 4;    //端口
}

